yarn-project/end-to-end/src/e2e_block_building.test.ts

```diff
@@ -415,29 +414,6 @@ describe('e2e_block_building', () => {
 
     afterAll(() => teardown());
 
-    it('calls a method with nested note encrypted logs', async () => {
```

Comment:
This is the no-longer necessary test. The test case below this one tests ordering of logs sufficiently and the protocol no longer cares about whether a private log is a note log or non-note log.